package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cordova-plugin-file-transfer",
-  "version": "1.7.1",
+  "version": "1.7.1-j5.1",
```

Comment:
This version is also in package-lock.json. I think maybe package-lock.json should be regenerated?

Reply:
There is no package-lock.json in this project

Reply:
Oh weird. It's only one the master branch. Ignore this then.